src/interfaces/scenario/model/actor/TestflightA380Ber.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace interfaces\scenario\model\actor;
+
+use interfaces\model\data\material\building\airport\AirportInterface;
+use interfaces\model\data\material\person\MajorInterface;
+use interfaces\model\data\collection\JournalistCollectionInterface;
+use interfaces\model\data\material\vehicle\car\AircraftTractorInterface;
+use interfaces\model\data\material\vehicle\plane\PlaneInterface;
+use interfaces\model\data\collection\GuestCollectionInterface;
+
+/**
+ *
+ * @author kevinfrantz
+ *        
+ */
+interface TestflightA380Ber extends ActorCollectionInterface
```

Comment:
Würdest du für einen Testflug in München ein weiteres Interface anlegen?

Reply:
Nö. Für die konkreten Szenarios wären keine Interfaces notwendig gewesen :)